service/javadsl/server/src/main/java/com/lightbend/lagom/javadsl/server/ServiceGuiceSupport.java

```diff
@@ -110,10 +113,30 @@ default void bindServices(ServiceBinding<?>... serviceBindings) {
     }
 
     /**
+     * Binds a Service interface with its implementation and registers it for publishing.
+     * <p>
+     * Inspects all bindings and creates routes to serve every call described in the bound services.
+     * <p>
+     * Builds the {@link ServiceInfo} metadata using only the <code>locatable</code> services.
+     *
+     * @param serviceBinding a {@link ServiceBinding}. Use the convenience methods
+     *                       {@link ServiceGuiceSupport#serviceBinding(Class, Class)} to build the
+     *                       {@link ServiceBinding}s. If you are building a Lagom
+     *                       service that acts only as a consumer and doesn't need to bind any service you should
+     *                       not use {@link ServiceGuiceSupport#bindService(ServiceBinding)}  or
+     *                       {@link ServiceGuiceSupport#bindServices(ServiceBinding[])} and should use
+     *                       {@link ServiceGuiceSupport#bindServiceInfo(ServiceInfo)} instead.
+     */
+    default void bindService(ServiceBinding<?> serviceBinding) {
+        bindServices(serviceBinding);
+    }
```

Comment:
I originally tried to keep the change to a minimum. After you mentioned I've tried your suggestion  and I actually prefer how the end code looks like.